### DISCUSSION TITLE
Edit (and commit) MD files in the app

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#5708] [Mobile] [answered]
It would be handy if you could edit and commit Markdown files right in the app.
This way you could update your repository while you are on the go.

**Comment by fishooo0815** [comment]
Yes!!! I want that, too.

**Reply by fishooo0815** [reply]
🚀

**Comment by eliperkins** [comment, marked as answer]
Thanks for the feedback @Zerotask! Editing code is definitely on our radar, and we want to make sure that we do it right!

I've added your notes to our internal tracking issue about editing code within the app. I'll talk to the team about this some more!

**Comment by spaltrowitz** [comment]
thanks for your suggestion @Zerotask! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Reply by Zerotask** [reply]
Thank you for the update. Although I'm an Android user...
